math-libs/BLAS/CMakeLists.txt

```diff
@@ -158,8 +158,8 @@ if(THEROCK_ENABLE_SPARSE)
       -DHIP_PLATFORM=amd
       -DROCM_PATH=
       -DROCM_DIR=
-      -DBUILD_CLIENTS_SAMPLES=OFF
-      -DBUILD_CLIENTS_BENCHMARKS=OFF
+      -DBUILD_CLIENTS_TESTS=ON
+      -DBUILD_CLIENTS_BENCHMARKS=ON
```

Comment:
I wonder if we should add a `THE_ROCK_ENABLE_SUBPROJECT_TESTS` or forward variables like `HIPSPARSE_BUILD_CLIENTS_TESTS` to control these, rather than always enable. WDYT?

Reply:
Generally speaking, yes.However, with this we would not only introduce a new flag but increase complexity a little. Beside such a flag there are dependencies which we only need when these flags are enabled. Furthermore, the RPATH can only be set on existing targets. Both not a big deal. What _could_ be more complicated (I haven't looked into it) is the artifact generation. So far I didn't picked it up as it might not be of a high important for the CI whereas for a user it is definitely desirable. Any thoughts @stellaraccident? Happy to fill an issue to follow up.